packages/settingregistry/src/tokens.ts

```diff
@@ -173,6 +173,89 @@ export namespace ISettingRegistry {
     | 'object'
     | 'string';
 
+  /**
+   * Menu defined by a specific plugin
+   */
+  export interface IMenu extends PartialJSONObject {
+    /**
+     * Unique menu identifier
+     *
+     * #### Notes
+     * Standard menu ids are: jp-menu-file, jp-menu-edit, jp-menu-help, jp-menu-kernel,
```

Comment:
perhaps hoist out of comment into a type, and/or combined with some piece of data:

```ts

export const DefaultMenuPriority = {
  'jp-menu-file': 1,
  'jp-menu-edit': 2,
  // ...
}

export type TDefaultMenuId = keyof typeof DefaultMenuPriority;

export interface IMenu extends PartialJSONObject {
   id: TDefaultMenuId | string
```